src/Panel.vue

```diff
@@ -71,48 +72,43 @@
     },
     props: {
       header: {
-        type: String
+        type: String,
+        default: ''
       },
       alt: {
-        type: String
+        type: String,
+        default: ''
       },
       type: {
         type: String,
         default: null
       },
       expandable: {
```

Comment:
'expandable === false' scenario: The panel should have been expanded when the page loaded.